src/rules/completedDocsRule.ts

```diff
@@ -39,6 +39,7 @@ export const ALL = "all";
 
 export const ARGUMENT_CLASSES = "classes";
 export const ARGUMENT_ENUMS = "enums";
+export const ARGUMENT_ENUM_MEMBERS = "enumMembers";
```

Comment:
Nit: other rules have options like `"check-accessor"`, `"member-variable-declaration"`, "`array-simple`", etc. This should probably be `"enum-member"`.